packages/react/test/specs/commonTests/isConformant.tsx

```diff
@@ -53,40 +58,40 @@ export default (Component, options: Conformant = {}) => {
 
   const componentType = typeof Component
 
-  // This is added because the component is mounted
+  const helperComponentNames = [
+    ...[ThemeProvider, FelaTheme, Ref, RefFindNode],
+    ...(usesWrapperSlot && [Box]),
+  ].map(getDisplayName)
+
+  const toNextNonTrivialChild = (from: ReactWrapper) => {
+    let current = from.childAt(0)
+
+    if (!current) return current
+
+    while (helperComponentNames.indexOf(current.name()) >= 0) {
```

Comment:
nit: may be recursion in this case will be more readable?

Reply:
lets try that :)